web/components/admin/Toolbar/index.js

```diff
@@ -10,10 +10,17 @@ class Toolbar extends React.Component {
       mentor: { value: 'mentor', label: 'Mentor' },
       volunteer: { value: 'volunteer', label: 'Volunteer' },
     };
+    this.sortSelectLabels = {
+      name: { value: 'name', label: 'Name' },
```

Comment:
Do you think we could make this `lastName: { value: 'lastName', label: 'Last Name' },`? As a user it wasn't immediately clear to me that we were using last name to sort. Also feel free to add a `firstName` sort filter if you make this change

Reply:
Good idea, I'll add in the firstName too